deploy-manage/manage-connectors.md

```diff
@@ -2,25 +2,98 @@
 mapped_urls:
   - https://www.elastic.co/guide/en/kibana/current/action-types.html
   - https://www.elastic.co/guide/en/serverless/current/action-connectors.html
+applies_to:
+  stack: ga all
+  deployment:
+    hosted: ga
+    ece: ga
+    eck: ga
+    self: ga
+  serverless:
+    security: ga
+    elasticsearch: ga
+    observability: ga
```

Comment:
we could go two directions with this:

deployment type focus
```suggestion
applies_to:
  deployment:
    hosted: ga
    ece: ga
    eck: ga
    self: ga
  serverless:
```

stack vs. serverless focus
```suggestion
applies_to:
  stack:
  serverless:
```

doing both is overkill

no need to list the serverless project types unless they're relevant (they are not here)